src/userCLI/scripts/common

```diff
@@ -9,6 +9,8 @@ else
     GIT_BRANCH=$TRAVIS_BRANCH
 fi
 
+TAG_SHA_OLD="mrcide-${PACKAGE_NAME}:${GIT_SHA}"
+TAG_SHA_NEW="${TAG_IMAGE}:${GIT_SHA}
```

Comment:
`TAG_SHA_NEW` here is the same as `TAG_SHA` so can be removed and `TAG_SHA` used instead [here](https://github.com/mrc-ide/hint/pull/727/files#diff-6b34fe22274512dc0f81f2c1a0e1bbe90af06d2d1d7ac0de6a28325a034adac8R12). But it looks like you're passing the desired tag name to the docker build task [here](https://github.com/mrc-ide/hint/pull/727/files#diff-5b27a5a792aaf4ebf48a1bb1d03cb1e08656570fa4ea796dcedfb193dccff214R51) - is that not working as expected?

Reply:
JetBrains tags Docker image with project name by default, I couldn’t rename from the setup. TAG_SHA is Exported, used across the app so I wanted to preserve the ARG. 

JetBrains image tag = `mrcide.hint-cli….`

We are expecting 
`tag = mrcide/hint-cli…`


So, retagging is just to meet our need.